testing/e2e/app/routes/Table/route.tsx

```diff
@@ -13,6 +13,26 @@ export default function Resizing() {
   const subRows = searchParams.get('subRows') === 'true';
   const filter = searchParams.get('filter') === 'true';
   const selectSubRows = !(searchParams.get('selectSubRows') === 'false');
+  const enableVirtualization = searchParams.get('virtualization') === 'true';
+  const empty = searchParams.get('empty') === 'true';
+  const scroll = searchParams.get('scroll') === 'true';
+  const oneRow = searchParams.get('oneRow') === 'true';
+
+  const virtualizedData = React.useMemo(() => {
+    const size = oneRow ? 1 : 100000;
+    const arr = new Array(size);
+    if (!empty) {
+      for (let i = 0; i < size; ++i) {
+        arr[i] = {
+          index: i,
+          name: `Name${i}`,
+          description: `Description${i}`,
+          id: i,
+        };
+      }
+    }
+    return arr;
+  }, []);
```

Comment:
This is missing `oneRow` and `empty` from the dependency array

Reply:
Added to the dependency array in this [commit](https://github.com/iTwin/iTwinUI/pull/2092/commits/2901d08b7970d1a6d2a40eb49b36be5badb79d44).